eng/scripts/Create-Apiview-Token-Files.ps1

```diff
@@ -0,0 +1,39 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true)]
+  [string]$ReviewDetailsJson,
+  [Parameter(Mandatory = $true)]
+  [string]$StagingPath,
+  [Parameter(Mandatory = $true)]
+  [string]$WorkingDir,
+  [Parameter(Mandatory = $true)]
+  [string]$StorageBaseUrl,
+  [Parameter(Mandatory = $true)]
+  [string]$ContainerSas,
+  [Parameter(Mandatory = $true)]
+  [string]$Language
+)
+
+
+Write-Host "Review Details Json: $($ReviewDetailsJson)"
+$reviews = ConvertFrom-Json $ReviewDetailsJson
+if ($reviews -ne $null)
+{
+    foreach($r in $reviews)
+    {
+        $codeDir = New-Item -Path $WorkingDir/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
+        $sourcePath = $StorageBaseUrl + "/" + $r.FileID + $ContainerSas
+        Write-Host "Copying $($sourcePath)"
+        azcopy cp "$sourcePath" $codeDir/$($r.FileName) --recursive=true
+
+        #Create staging path for review and revision ID
+        New-Item -Path $StagingPath/$($r.ReviewID)/$($r.RevisionID) -ItemType Directory
+        $languageScriptName = "./Create-Apiview-Token-$Language.ps1"
```

Comment:
We should just pass this inner script we want to call as a variable so that language specific implementation can just pass in the path to whatever script they want.

Reply:
done